### ISSUE TITLE
Overwritten and modified ODS files not synced

### ISSUE THREAD

**Issue by BBBalls** [hwittenborn/celeste#182]
When an LibreOffice Calc ODS file in the local sync folder is overwritten, an `directory not found` error is produced, and the overwriting ODS file is not synced to Proton Drive.

When an ODS file in the local sync folder is edited, an `directory not found` error is produced, and the modified ODS file version is not synced to Proton Drive.

For the  syncing of newer versions of ODS files with the same file name to occur, the original ODS file has to be deleted from Proton Drive.

**Comment by hwittenborn** [comment]
Hey @BBBalls! Are you still encountering this issue? I'm not entirely sure what might be causing it, there shouldn't be something specific to ODS files that'd be causing an issue. Do you experience the same behavior with other file types?

**Comment by BBBalls** [comment]
Yes, other file types seem to be affected. A test with a markdown file resulted in the same kind of error after editing it. The markdown file was synced from Proton Drive to the local folder, but after editing the file `directory not found` error was given, and the edited file was not synced with Proton Drive. The sync of the edited file only after the original file is deleted in Proton Drive.

This same phenomenon is occurring with the `.sync-exclude.lst` file as well. I just switched distro, so generated a new local sync folder. The unmodified .sync-exclude.lst file, as in blank, that is in the Proton Drive is in conflict with the newly generated and blank local .sync-exclude.lst file, giving the `directory not found` error.

My previous report was from Pop!_OS 22.04 using the Celeste Flatpak. The behavior was reproduced on Arch Linux using the Celeste Flatpak.
